docs/content/anonymization/core-anonymizers.md

```diff
@@ -254,6 +262,17 @@ customer:
 ```
 :::
 
+:::tip
+Hashing a string is not anonymizing it because hash functions have a reproducible output,
+this in order to avoid decrypting data using rainbow tables, per default a salt will be
+added to string values prior to hasing.
```

Comment:
```suggestion
Hashing a string is not anonymizing it because hash functions have a reproducible output.
In order to avoid decrypting data using rainbow tables, a salt will be added by default to
string values prior to hashing.
```